AcquisitionInfo/explainer.md

```diff
@@ -0,0 +1,223 @@
+at which point can be captured by the Acquisition Info API. _[See the Web Install API explainer for more information.][Web Install API]_
+
+## Privacy and Security Considerations
+
+1. _Lack of validation_: It is assumed that the Store platform does not perform any validation
+  or sanitation of the Campaign ID itself.
+
+2. _Lack of user control regarding acquisition identifiers_: This solution provides no method for
+   the user to clear, disable, or modify the data retrieved from `acquisitionInfoProvider`.  We recognize that different vendors may have specific preferences and requirements regarding this aspect,
+   so we defer to the individual Store platforms to handle that implementation.
+
+3. _Potential misuse of acquisition identifiers_: Acquisition identifiers could potentially be
+   manipulated or obscured by ad networks or referral sources, similar to the existing practices
+  in advertising. However, this behavior is not specific to the proposed API and is already
+  possible with ad campaigns.
+
+This API can only be invoked in a top-level [secure context][secure context]. This is in order to prevent attackers from accessing privacy information that
+is being exposed in this API.
+
+## Glossary
+
+| Term        | Definition                                                          |
+| ----------- | ------------------------------------------------------------------- |
+| Attribution | Identifying the source by which an installation occured.            |
+| Acquisition | Installation of the application in question.                        |
+| PWA         | Progressive Web App                                                 |
+| UWP         | Universal Windows Platform                                          |
+| API         | Application Programming Interface                                   |
+| WinRT       | Windows Runtime                                                     |
+| Store       | Microsoft Store                                                     |
+| App         | Application                                                         |
+| Campaign ID | A unique identifier that is associated with a specific ad campaign. |
+
+[WinRT API]: https://learn.microsoft.com/en-us/uwp/api/windows.services.store.storecollectiondata.campaignid
+[display mode]: https://developer.mozilla.org/en-US/docs/Web/Manifest/display
+[Play Install Referrer API]: https://developer.android.com/google/play/installreferrer
+[Apple Ads Attribution API]: https://developer.apple.com/documentation/ad_services
+[secure context]: https://w3c.github.io/webappsec-secure-contexts/
+[Web Install API]: https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/WebInstall/explainer.md
+
+## Open Questions
+
+### Should we be returning some sort of install source on a browser installed app query?
+
+In the [current iteration of this proposal](#same-domain-installation), the API returns an empty string in the `storePlatform` field
```

Comment:
I think the callout here for Policy-installed web apps is a good one.  I'd propose elevating policy-installed web apps to another sub-section above, and then just use `installSource` = `Policy` as its value.  Maybe it could additionally have an install date/time stamp.